### DISCUSSION TITLE
Multibanking FinTS

### DISCUSSION THREAD

**Discussion by Absolutenovice** [mtotschnig/MyExpenses#1430] [General]
Hi!
I love the app, its great!
I added my DKB account in multibanking. My account now appears in Multibanking. But when I select "Accounts" and enter my password, it just shows the loading forever. I dont have Google services on my phone, might have something to do with that?

Greetings!

**Comment by mtotschnig** [comment]
@Absolutenovice Thanks for the report. Which version from which app store are you using?

**Comment by Absolutenovice** [comment]
Hi!
I am using Version 3.7.5 from F-Droid.

**Comment by mtotschnig** [comment]
I have tested connecting to DKB on an emulator without Google services, and did not encounter any problems. Could you maybe try to remove the bank, and then add it again and if you are prompted for choosing a TAN media, select a different one. In my case, I see "pushTAN" and my phone listed. Also it might be helpful, if you could switch on "Write debugging information into system log", restart the app, and after using the Banking feature share "Banking-latest.log".

**Comment by Absolutenovice** [comment]
It works now, the problem was on my end. The issue was the TAN media.
Thank you
